functional_tests/steps/article_page.py

```diff
@@ -0,0 +1,10 @@
+from behave import given
+from behave.runner import Context
+
+from cms.articles.tests.factories import ArticleSeriesFactory, StatisticalArticlePageFactory
+
+
+@given("a statistical article page has been published under the existing theme page")
+def create_article_page(context: Context):
+    context.article_series = ArticleSeriesFactory(parent=context.topic_page)
+    context.statistical_article = StatisticalArticlePageFactory(parent=context.article_series)
```

Comment:
@kacperpONS worth noting I have a similar one in #96 - https://github.com/ONSdigital/dis-wagtail/pull/96/files#diff-326f13bd4cb7217b4cf1b63f44d0e10ec184fb2310ddd8efb83af4ab61198e95R1-R15

Reply:
I'll change the wording wording to avoid conflicts later on. 

One thing to note, you don't need to pass the title to the article page factory as it gets automatically populated by Faker. It will then be referenced with `context.article.title` so having the hard-coded value there doesn't add much value.

Reply:
And a question - it feels like the two sequences of steps:
1. Create theme page -> create topic page
2. Create article series -> create an article

will be used *almost always* together. 

How about we use the specific steps only when it's necessary and for scenarios where they're used together define steps that will perform the whole sequence to avoid duplication?

Reply:
> One thing to note, you don't need to pass the title to the article page factory as it gets automatically populated by Faker. It will then be referenced with context.article.title so having the hard-coded value there doesn't add much value.

I am well aware of that. In some cases I do want these to be deterministic

Reply:
> How about we use the specific steps only when it's necessary and for scenarios where they're used together define steps that will perform the whole sequence to avoid duplication?

100% on this with you.